package.json

```diff
@@ -31,7 +31,8 @@
     "umd-shopify-buy:minify": "uglifyjs --compress --mangle -o dist/shopify-buy.umd.min.js -- dist/shopify-buy.umd.js",
     "umd-shopify-buy-polyfilled:build": "rollup -f cjs --config .rollup.config.umd.polyfilled.js src/shopify-polyfilled.js | node scripts/runBabel umd > dist/shopify-buy.umd.polyfilled.js",
     "umd-shopify-buy-polyfilled:minify": "uglifyjs --compress --mangle -o dist/shopify-buy.umd.polyfilled.min.js -- dist/shopify-buy.umd.polyfilled.js",
-    "commonjs:build": "cross-env BABEL_ENV=commonjs VERSION=$(get-project-version --template v{{version}}-{{commit}}) babel --ignore src/polyfilled.js -d lib/ src/",
+    "commonjs:build": "rm -rf lib; cross-env BABEL_ENV=commonjs VERSION=$(get-project-version --template v{{version}}-{{commit}}) babel --ignore src/polyfilled.js -d lib/ src/",
+    "jsnext:build": "rm -rf lib-jsnext; cross-env BABEL_ENV=jsnext VERSION=$(get-project-version --template v{{version}}-{{commit}}) babel --ignore src/polyfilled.js -d lib-jsnext/ src/",
```

Comment:
Maybe we should disable tree shaking and see how big of an optimization tree shaking brings. Because we don't have that many external dependencies I wonder what the impact would be.

The weird export/import issue's we've had have been in our own codebase/transpilation and not in consumption of the `js-buy-sdk`. This being said people can do "bad" things currently and yes in theory it does narrow the support footprint.

Doing this will definitely be a major bump as it could break any module that may be importing something other than `shopify.js`.

I'm trying to think of other reasons to not do this but I can't find good reasons so let's go with it and revert later if need be. @tessalt do you see any issues?